content/docs/introduction/getting_started.md

```diff
@@ -160,6 +160,7 @@ processes:
 # Fetch the client library code and compile example.
 git clone https://github.com/prometheus/client_golang.git
 cd client_golang/examples/random
+go get
```

Comment:
As a user with a proper go setup, I don't want example binaries in my $PATH.

On Fri, Aug 28, 2015, 7:05 PM Björn Rabenstein <notifications@github.com>
wrote:

> In content/docs/introduction/getting_started.md
> <https://github.com/prometheus/docs/pull/195#discussion_r38221082>:
>
> > @@ -160,6 +160,7 @@ processes:
> >  # Fetch the client library code and compile example.
> >  git clone https://github.com/prometheus/client_golang.git
> >  cd client_golang/examples/random
> > +go get
>
> Hmm, shouldn't we assume the user has a proper Go setup? Then it would
> really just be go get.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/prometheus/docs/pull/195/files#r38221082>.
>